### PR TITLE
Update where filter in readme

### PR DESCRIPTION
**Pull request by ssh24**
Update readme to fix nested level `and` filter. 

connect to https://github.com/strongloop/loopback-datasource-juggler/issues/1327

### CODE REVIEW

**Review by b-admike — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The output would be:
 applyFilter({
   where: {
     // the price > 10 && price < 100
-    price: {gt: 10, lt: 100},
+    and: [{id: {gt:10}}, {id:{lt:100}}],
```
Comment (dhmlau) [inline README.md]: any reason we change from price to id? 
from the comment, it's also about `price`.

**Review by jannyHou — APPROVED** [review]
~lgtm~ Sorry, same question as Diana :p

File: README.md (hunk 1 above)
Comment (kjdelisle) [inline README.md]: Can we break this out into multiple lines?
```
and: [
  {
    id: { gt: 10 }
  },
  {
    id: { lt: 100 }
  },
],
```

**Review by kjdelisle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kjdelisle) [reply README.md]: Yeah, the ID field isn't what we want to filter on.

Comment (0candy) [inline README.md]: Would it not be `price`? Why is it `id`?

Comment (0candy) [reply README.md]: Sorry, same comment as above.

Comment (ssh24) [reply README.md]: Right. Sorry for that. It is a typo -_-

**Review by b-admike — APPROVED** [review]

**Review by kjdelisle — APPROVED** [review]

**Review by 0candy — APPROVED** [review]

**Review by dhmlau — APPROVED** [review]

**Review by jannyHou — APPROVED** [review]

**Pull request merged**